bingo/bingo-elastic/python/bingo_elastic/elastic.py

```diff
@@ -333,20 +337,19 @@ def filter(
         )
         res = self.el_client.search(index=self.index_name, body=query)
         yield from response_to_records(
-            res, self.index_name, postprocess_actions
+            res, self.index_name, postprocess_actions, indigo_session, options
         )
 
 
 # pylint: disable=too-many-arguments
 def compile_query(
     similarity: BaseMatch = None,
     exact: IndigoRecord = None,
-    substructure: IndigoRecord = None,
-    limit: int = 10,
+    substructure: IndigoObject = None,
+    limit: int = 5000,
```

Comment:
Probably 100 as better as default value

Reply:
@AATDev21 @mkviatkovskii same as before. Default Elastic one is 10. We could override if needed. Getting 1 page of results of size 5000 is big load. Not sure for 100 too. Paging is much better way to deal with it

Reply:
In some test cases like substructure search for an empty query molecule, we expected to get 2026 hits, and a low limit value will lead this test to fail. I really shouldn't have changed limit's default value, so let's set it back to Elastic's default 10.